### ISSUE TITLE
Add Release Candidate Banner

### ISSUE THREAD

**Issue by melcorr** [theforeman/foreman-documentation#695]
Just as @upadhyeammit is doing https://github.com/theforeman/theforeman.org/pull/1887
We need to have an RC banner.

**Comment by lzap** [comment]
Yeah I agree, the banner should be rewritten in a way we can put any text in there. Another candidate would be: Unsupported for very old releases.

**Comment by lzap** [comment]
Implemented in https://github.com/theforeman/foreman-documentation/pull/710

**Comment by ekohl** [comment]
This is still open issue. The goal of this banner is to inform users that the documentation they're reading documentation for a version that's not intended for production.

**Comment by asteflova** [comment]
Where exactly should the banner be shown? In every guide published for a version that's not intended for production?

The version dropdown currently includes nightly/supported/unsupported:
![Screenshot From 2025-01-10 15-19-46](https://github.com/user-attachments/assets/ecb289c5-247e-42a0-9708-d87520d72e28)
I take it we're looking for something more than this? That "nightly/unsupported" doesn't automatically mean "not intended for production"?

**Comment by maximiliankolb** [comment]
I had something as follows in mind saying "Update now":

![image](https://github.com/user-attachments/assets/f1dce406-3b40-4bd3-b767-1e103965b033)

source: https://simonwhitaker.github.io/github-fork-ribbon-css/

**Comment by asteflova** [comment]
To summarize what we want:

* A banner that says "Pre-release version", displayed in each guide for nightly and release candidate
* A banner that says "Unsupported version", displayed in each guide for an unsupported release. This should also link to somewhere useful (perhaps simply just to the [landing page](https://docs.theforeman.org/) which explains the context of supported/unsupported versions?)

**Comment by asteflova** [comment]
A possible solution that is based on pure asciidoc is to add a section to master.adoc, but include it only for pre-release/unsupported versions (with section numbering disabled). A section like that would be displayed at the beginning of each guide + in the ToC, essentially working as a banner of sorts.

A fancier solution is to add a ribbon or a banner in the header (anything that stays in view and doesn't disappear with scrolling) but there I don't know how to make it show only for the pre-release and unsupported versions.

Is there anyone who might be able to take a stab at this? Or do you think it's good enough if I just go with the simple asciidoc solution?

**Comment by Lennonka** [comment]
You have to cherry pick it to the other branches as well to make it work for those releases.
